OutOfSchool/Tests/OutOfSchool.Tests.Common/TestDataGenerators/ProviderDtoGenerator.cs

```diff
@@ -27,7 +27,8 @@ public static class ProviderDtoGenerator
             .RuleFor(x => x.Status, f => f.Random.Bool())
             .RuleFor(x => x.UserId, f => f.Random.Guid().ToString())
             .RuleFor(x => x.LegalAddress, _ => AddressDtoGenerator.Generate())
-            .RuleFor(x => x.ActualAddress, _ => AddressDtoGenerator.Generate());
+            .RuleFor(x => x.ActualAddress, _ => AddressDtoGenerator.Generate())
+            .RuleFor(x => x.InstitutionType, f => f.Random.ArrayElement((InstitutionType[])Enum.GetValues(typeof(InstitutionType))));
```

Comment:
Couldn't `PickRandom` be used instead of `f.Random.ArrayElement((InstitutionType[])Enum.GetValues(typeof(InstitutionType)))` [see SO](https://stackoverflow.com/questions/52819587/bogus-faker-how-to-pick-a-random-enum-value)?

Reply:
Same in `ProvidersGenerator`

Reply:
Thanks for the information. I did not know it. Fixed.